docs/contributing/999-rfcs/2024-06-26-wing-docs-v2.md

```diff
@@ -0,0 +1,161 @@
+| How do they learn? | Benefits from how-to guides and examples. How to guides include things like deploying websites, deploying resources into the cloud, focused on business logic. | Benefits from how-to guides and examples. More focused on how to create and maintain libs that others can use or examples of creating custom platforms. | Benefits from how-to guides, either deploying applications into the cloud or creating libs and platforms for others. |
+| How do they interact with wing? | Consumer of wing standard libs, custom wing libs and custom platforms. | Produces wing libs and custom platforms for others to consume. | Can be both the consumer of wing and author of libs. |
+| Testing | Needs to understand how to test wing applications | Needs to understand how to test wing applications and libs. | Needs to understand how to test wing applications and libs. |
+| Programming langugages | May be interested in multi-lang preflight syntax | May be interested in multi-lang preflight syntax  | May be interested in multi-lang preflight syntax  |
+| What level of understanding do they need with wing? | Needs to understand the core concepts of wing and may need to the ability to dive deeper into wing. | Needs to understand the core concepts of wing and deeper dive content. | Needs to understand the core concepts of wing and deeper dive content
+
+Each of these personas have a different need for wing documentation and may have different entry points within the docs. Based on these personas, the proposed solution is to organise existing content, introduce new sections/pages, add more automation into docs and develop a new patterns/examples page.
+
+## Sitemap proposal
+
+The sitemap below visually explains the proposed solution for docs v2. 
+
+
+[Open the sitemap](https://drive.google.com/file/d/1jsZ4cXpf2v8Au3qPTJsU50PtK0zV5rqZ/view?usp=sharing)
+
+Content has been marked with tags:
+
+- **exists**: content already exists
+- **multi-lang:** content needs to support multi-lang examples (TypeScript and Wing)
+- **video**: content can/should support video content in the future
+- **interactive-tutorial**: make the content interactive (with playground)
+- **note**: just a note
+
+### **Install section**
+
+Simple link in the navigation bar that will take users to install wing.
+
+### Learn section
+
+The learn section is for people that want to learn wing (currently “docs”). This section is designed for users that want to learn wing, initially going through basics and fundamentals into progressively deeper content (i.e wing libs, platforms, wing fundamentals). Users will initially learn core concepts of wing through the tutorials in the getting started section. This will expose them to concepts of wing including inflight/preflight, winglibs, testing and deploying.
```

Comment:
```suggestion
The learn section is for people that want to learn wing (currently “docs”), initially going through basics and fundamentals into progressively deeper content (i.e wing libs, platforms, wing fundamentals). Users will initially learn core concepts of wing through the tutorials in the getting started section. This will expose them to concepts of wing including inflight/preflight, winglibs, testing and deploying.
```